pdc_client/plugins/component.py

```diff
@@ -104,13 +112,17 @@ def global_component_create(self, args):
 
     def global_component_update(self, args):
         data = extract_arguments(args)
+        global_component_id = self.get_component_id(args.global_component_name)
+        if not global_component_id:
+            sys.stderr.write("This global component don't exist")
```

Comment:
'don't exist' should be 'doesn't exit'?
Also it's better add new line in sys.stderr.write.